<ui version="4.0" >
 <class>weatherAddPlace</class>
 <widget class="QWidget" name="weatherAddPlace" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>243</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Found Places</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Found places</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="foundPlacesListView" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
